internal/page/certificateprovider/confirm_your_details.go

```diff
@@ -47,6 +48,11 @@ func ConfirmYourDetails(tmpl template.Template, donorStore DonorStore, certifica
 			App:                 appData,
 			CertificateProvider: certificateProvider,
 			Donor:               donor,
+			PhoneNumberLabel:    "mobileNumber",
+		}
+
+		if certificateProvider.DonorChannel.IsPaper() {
```

Comment:
I think there was a simpler implementation than passing this everywhere, we know the donor is on paper if `donorStore.GetAny(ctx)` returns a `dynamo.NotFoundError{}`. That would then also save us having to rework as much later